Projects/Server/Geometry/Point3DList.cs

```diff
@@ -9,7 +9,7 @@ public class Point3DList
 
     public Point3DList()
     {
-        m_List = new Point3D[8];
+        m_List = new Point3D[16];
```

Comment:
Bumped this to 16 because `eye` level is 14, and those checks use 15 point locations, so it is pretty common.